core-primitives/randomness/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
Hmm, didn't I already have this implemented?

Reply:
you did in the raffle PR and I took it from there. but it is now a standalone crate